Security/TwoFactor/Provider/TwoFactorProviderRegistry.php

```diff
@@ -24,6 +27,20 @@ class TwoFactorProviderRegistry implements AuthenticationHandlerInterface
     private $providers;
 
     /**
+     * Event dispatcher.
+     *
+     * @var EventDispatcherInterface
+     */
+    protected $eventDispatcher;
+
+    /**
+     * Request parameter name used for code.
+     *
+     * @var string
+     */
+    protected $authRequestParameter = '_auth_code';
```

Comment:
Please use private.

No need to pre-set value when constructor injection is used.